### DISCUSSION TITLE
not working?

### DISCUSSION THREAD

**Discussion by spinalcode** [valinet/ExplorerPatcher#2603] [Q&A]
I noticed that when installing this on the latest Windows 11 update, it no longer works. has Microsoft purposely patched it out?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Comment by spinalcode** [comment]
Windows 11 Home
Version 23H2
OS Build 22631.2861
Experience Pack 1000.22681.1000.0

I have no other software that mod the UI yet, and there are no changes made to the properties, because it does not load.
Once setup has installed the application, explorer restarts, with no changes made, there is no settings icon in the notification area either.



[edit] I seem to have got it working again, I uninstalled, then removed any traces of it from regedit, then reinstalled.

**Reply by pyrates999** [reply]
Please close this then.
